notebooks/en/issues_in_text_dataset.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    os.environ["TOKENIZERS_PARALLELISM"] = "false"  # disable parallelism to avoid deadlocks with huggingface
</pre><p>Is this needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>